### ISSUE TITLE
AddRange pattern that panics

### ISSUE THREAD

**Issue by anacrolix** [RoaringBitmap/roaring#53]
This sequence of calls causes a panic on linux/arm and darwin/amd64

```
bm := NewRoaringBitmap()
bm.AddRange(21, 26)
bm.AddRange(9, 14)
bm.AddRange(11, 16)
```

**Comment by anacrolix** [comment]
The 3rd call:

goroutine 2861759 [running]:
github.com/RoaringBitmap/roaring.(*arrayContainer).iaddRange(0x16cc0c40, 0xb, 0x10, 0x0, 0x0)
        /root/gopath/src/github.com/RoaringBitmap/roaring/arraycontainer.go:101 +0x3ec
github.com/RoaringBitmap/roaring.(*RoaringBitmap).AddRange(0x158668d0, 0xb, 0x10)
        /root/gopath/src/github.com/RoaringBitmap/roaring/roaring.go:878 +0x1dc

**Comment by lemire** [comment]
Investigating. The AddRange function is relatively recent so bugs are possible.

**Comment by lemire** [comment]
Fixed in version 0.1.1. Please upgrade.

Closing. Reopen if you cannot verify.
